### DISCUSSION TITLE
Set how aggressive rockets are?

### DISCUSSION THREAD

**Discussion by pierrelasse** [YimMenu/YimMenu#2119] [Q&A]
Is it possible to set how aggressively rockets target from your vehicle in a script?

**Comment by binded-zz** [comment]
there is about to be a merge that will add such options as well as many more 
https://github.com/YimMenu/YimMenu/pull/2081

hope soon 
i been messing with the buildbot test build and its works great 
i set a high timeout and long range lock and range and reduce time to shot  
its like a rocket cannon cars go  flying into the air over low buildings subs come out of the ocean
change to script large explotions or  orbital cannon  LOL

its amazing
